sycl/include/CL/sycl/detail/pi.h

```diff
@@ -1219,9 +1220,12 @@ piextProgramGetNativeHandle(pi_program program, pi_native_handle *nativeHandle);
 ///
 /// \param nativeHandle is the native handle to create PI program from.
 /// \param context is the PI context of the program.
+/// \param ownNativeHandle tells if SYCL RT should assume the ownership of
```

Comment:
This all looks good, but IMO it might be improved if the bool variable name was more explicit, informing us _whose_ ownership is being represented.   As it is now it could be ambiguous.    Maybe `syclRTOwnsNativeHandle`  would be better, or `syclPluginOwnsNativeHandle` ?

Reply:
Isn't the comment clarifying this enough: "tells if SYCL RT should assume the ownership"?

Reply:
yes, but as the various plugins start actually implementing this it's likely just the variable name that'll get copied into each individual implementation, not the comment.  And the interpretation of `ownNativeHandle` might be misinterpreted if you take the view of the caller vs the callee.  But it's minor, and you are right, the comment is very clear.

Reply:
OK, I've changed the name